### ISSUE TITLE
Error unmarshalling data while filtering for CVEs more than 120 days old

### ISSUE THREAD

**Issue by DreyAnd** [Un4gi/fave#1]
Example:
```
$ fave -k Apache -fy 1                                                                                   
> Error unmarshalling data: unexpected end of JSON input
unexpected end of JSON input
```

**Comment by Un4gi** [comment]
This is an issue with a recent change to NIST's API. Working on a fix, thanks for pointing this out!

**Comment by Un4gi** [comment]
As a temporary workaround, only use the `-fd` option to filter between 1-120 days (120 is the maximum number of days the NIST API will allow. A more permanent solution is still in the works.

**Comment by Un4gi** [comment]
Temporarily fixed - see [v1.0.1](https://github.com/Un4gi/fave/releases/tag/v1.0.1). Closing issue, but I will try to circumvent the NIST restriction to bring back filter-by-year and filter-by-month in a future release.

**Comment by DreyAnd** [comment]
Thanks man! Appriciated :)
